scripts/spec.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
What is this file and how is it used?
I don't see it mentioned anywhere and background image is also random and doesn't match window size specifiedin this file.

Reply:
This is config file for appdmg command to generate dmg. The size is actually Finder window size (not the application window size)